homeassistant/components/mastodon/strings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]",
+          "base_url": "[%key:common::config_flow::data::url%]",
+          "client_id": "Client Key",
+          "client_secret": "Client Secret",
+          "access_token": "[%key:common::config_flow::data::access_token%]"
+        },
+        "data_description": {
+          "base_url": "The URL of your Mastodon instance."
+        }
+      }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
+    },
+    "error": {
+      "credential_error": "The credentials are incorrect."
+    }
+  },
+  "issues": {
+    "deprecated_yaml": {
+      "title": "Mastodon YAML notify configuration imported",
+      "description": "Configuring Mastodon notify using YAML is being removed.\nRemove the YAML notify configuration and restart Home Assistant."
+    },
+    "import_aborted": {
+      "title": "Import aborted",
+      "description": "Configuring Mastodon using YAML is being removed.\nThe import was aborted, due to an existing config entry being the same as the data being imported in the YAML. Remove the YAML notify configuration and restart Home Assistant. Please use the UI to configure Mastodon."
+    },
+    "import_failed": {
+      "title": "Failed to import",
+      "description": "Failed to import the configuration for Mastodon.\nPlease use the UI to configure Mastodon. Don't forget to delete the YAML configuration."
+    }
+  }
```

Comment:
We should raise more specific issues for why the import failed according to ADR-0021. So if the username password is incorrect we should raise a different issue than if the host is wrong. 

(Note: if possible, we don't have to create hoops we can't jump through)